dashboard/requirements.txt

```diff
@@ -0,0 +1,11 @@
+requests
+pylint
+pytest
+pytest-cov
+pandas
+python-dotenv
+boto3
+pymssql
+streamlit
+altair
+sqlalchemy
```

Comment:
some packages like boto3 and some of the sql ones probably won't be used in the dashboard (e.g. boto3 is often used to work with AWS S3).

Might be a better idea to start with minimal ackages and add only those we need to keep our dashboard image light later on